### PR TITLE
Promisify and support Node.js >=4

### PR DESCRIPTION
**Pull request by kevva**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,46 +1,24 @@
...
-			return;
-		}
-
-		cb(null, obj);
+	return Promise.all([osxAppVersion(app), bundleId(app), appPath(app), appSize(app)]).then(res => {
+		return {
+			version: res[0],
+			bundle: res[1],
+			path: res[2],
+			size: res[3]
```
Comment (gillstrom) [inline index.js:21]: Maybe we should sort these alphabetically?

Comment (kevva) [inline index.js:21]: Hmm, I rather use the following:
1. `path`
2. `version`
3. `bundle`
4. `size`

But in reality, it doesn't matter :D. Maybe for the CLI though.

Comment (gillstrom) [inline index.js:21]: Yeah, I was mostly thinking about the CLI and some code style. But whatever, I'll merge :---)

**Comment by gillstrom** [comment]
Thanks! 💯

File: index.js (hunk 1 above)
Comment (kevva) [inline index.js:21]: We should change it. Makes more sense for CLI usage.

Comment (gillstrom) [inline index.js:21]: https://github.com/gillstrom/osx-app/commit/b57b340ac0bcc369156fcfe3295dc5e4a54aad55
:dancer:

**Pull request merged**